scripts/run_server

```diff
@@ -66,16 +76,12 @@ if [[ -n ${EXAMPLE+x} ]]; then
       echo "Invalid example variant: ${language}"
       exit 1;;
   esac
-fi
 
-declare -a ADDITIONAL_ARGS
-ADDITIONAL_ARGS+=(
-  "--grpc-tls-private-key=${SCRIPTS_DIR}/../examples/certs/local/local.key"
-  "--grpc-tls-certificate=${SCRIPTS_DIR}/../examples/certs/local/local.pem"
-  "--root-tls-certificate=${SCRIPTS_DIR}/../examples/certs/local/ca.pem"
-)
-if [[ -n "${OIDC_CLIENT+x}" ]]; then
-  readonly ADDITIONAL_ARGS+=("--oidc-client=${OIDC_CLIENT}")
+  if [[ "${EXAMPLE}" == "private_information_retrieval" ]]; then
```

Comment:
Although this works for a single example, this is not maintainable if multiple examples start using different config files. We should thinkg about a different way of passing this into the script.

Reply:
If only we could use a real programming language to run the examples... https://github.com/project-oak/oak/tree/master/runner :)